src/transaction/log_applier.c

```diff
@@ -7043,142 +7043,152 @@ la_print_log_arv_header (const char *database_name, LOG_ARV_HEADER * hdr, bool v
 }
 
 /*
- * la_log_page_check() - test the transaction log
- *   return: void
- *   log_path: log path
- *   page_num: test page number
+ * applyinfo_applied_log_info() - 
+ *   return: 
  */
 int
-la_log_page_check (const char *database_name, const char *log_path, INT64 page_num, bool check_applied_info,
-		   bool check_copied_info, bool check_replica_info, bool verbose, LOG_LSA * copied_eof_lsa,
-		   LOG_LSA * copied_append_lsa, LOG_LSA * applied_final_lsa)
+la_applyinfo_applied_log_info (const char *database_name, const char *log_path, bool check_replica_info, bool verbose,
+			       LOG_LSA * applied_final_lsa)
 {
   int error = NO_ERROR;
   int res;
-  char *atchar;
-  char active_log_path[PATH_MAX];
   char *replica_time_bound_str;
 
   assert (database_name != NULL);
   assert (log_path != NULL);
 
-  atchar = (char *) strchr (database_name, '@');
-  if (atchar)
-    {
-      *atchar = '\0';
-    }
+  LA_HA_APPLY_INFO ha_apply_info;
+  char timebuf[1024];
 
-  /* init la_Info */
   la_init (log_path, 0);
+  la_init_ha_apply_info (&ha_apply_info);
 
-  if (check_applied_info)
+  res = la_get_ha_apply_info (log_path, database_name, &ha_apply_info);
+  if ((res <= 0) || (ha_apply_info.creation_time.date == 0 && ha_apply_info.creation_time.time == 0))
     {
-      LA_HA_APPLY_INFO ha_apply_info;
-      char timebuf[1024];
-
-      la_init_ha_apply_info (&ha_apply_info);
-
-      res = la_get_ha_apply_info (log_path, database_name, &ha_apply_info);
-      if ((res <= 0) || (ha_apply_info.creation_time.date == 0 && ha_apply_info.creation_time.time == 0))
+      error = res;
+      if ((res < 0) && (ha_apply_info.creation_time.date != 0 || ha_apply_info.creation_time.time != 0))
 	{
-	  error = res;
-
 	  goto check_applied_info_end;
 	}
+      else
+	{
+	  printf ("\nERROR : Query object is not loaded\n\n");
+	  return ER_FAILED;
```

Comment:
or `goto check_applied_info_end;`

Reply:
I chose the 'hard coding' because there is not suitable and related error message with this issue in the 'cubrid.msg'

Reply:
There is no related with no "query results".
As you mentioned this [comment](https://github.com/CUBRID/cubrid/pull/3374/files#r851129610), generic error is not enough information

Reply:
What I meant to say was that, 
1. add error code and error message, then return error. Not looking for an existing error code for this situation.
or 
2. for the consistency, just goto `check_applied_info_end`

Reply:
Let's summarize your comment.

1. error message should be enough information to users.
(Not now there is no adequate error message)
2. Using `check_applied_info_end` goto phase means use the er_set().
3. or add error code and return error message.

Consequently, do you want not to use hard coding error message and add error message on `cubrid.msg`?

Reply:
No. The most recent comment was a summary of the comments I had made so far.

Add error code and error message for it 
or
just goto check_applied_info_end

because, In other error cases, they just go to `check_applied_info_end` not returning an error. 

if you want to set and return a specific error code, you need `er_set()` to log the error.

However, my opinion is that a `goto check_applied_info_end` is sufficient if you are going to handle it like any other exception.

Also, hard-coded error messages don't convey enough information to the user. It would be nice if you could tell the user why the error occurred.

Reply:
Oh, I totally understand. 

I can agree using `goto check_applied_info_end` phase :)

But still I have a question about using `er_set()`. 

1. Because  `r == 0` is not error case for `la_get_ha_apply_info`. But accroding to `applyinfo` utility, this is error case. 

2. I am not sure that addition new error message in the **cubrid.msg** is efficient choice.

So, these make me hesitate to use `er_set()`. What do you think?

Reply:
1. If it is an error case only for the utility, then you can add an error message for utility (utils.msg). 
2. Could you please tell me why you think it's not efficient?
3. So why are you trying to output an error message with hard-coding? if you want to print out error message with hard-coding, then you can use ER_HA_GENERIC_ERROR, as it is used in `la_get_applyinfo_copied_log_info`

Reply:
1. Oh, I didn't know that thank you so much!
2. I am not sure that does the message can be added whenever it is needed to use in ? I think it is too narrow to use this error message. So I think it is not efficient.
3. Because I cannot convince above 2nd reason.  **With** `ER_HA_GENERIC_ERROR` is good suggestion. Thank you!

Can I fix the message after you answer the 2nd question to convince about it? I thought about it what I can do the best, but it is not enough :( 
Could you please answer in the morning? I don't want to disturb your rest.